### PR TITLE
📆 2023 Schedule (not published yet)

### PR DESCRIPTION
**Pull request by drewbrew**
## Description

Create the 2023 schedule. It's not accessible from the site yet, but I'm going to merge this so that I can send this out to speakers to review.

### CODE REVIEW

File: _pages/schedule.html [hunk 1]
```diff
@@ -10,15 +10,16 @@
 
 <p>All sessions are in Pacific Daylight Time (UTC-07:00).</p>
```
Comment (jefftriplett) [inline _pages/schedule.html:11]: Just noting time zone change.

File: _pages/schedule.html [hunk 2]
```diff
@@ -10,15 +10,16 @@
...
 
-<p>The talks in Salon F-H are restricted in capacity due to COVID-19, so get your seat early! If there is not an available chair, you will not be allowed in the room.</p>
+<p>
+  The talks in Salon F-H are restricted in capacity due to COVID-19, so get your
```
Comment (jefftriplett) [inline _pages/schedule.html:14]: Noting location change.

**Review by jefftriplett — APPROVED** [review]

File: _posts/2023-07-10-announcing-lineup.md [hunk 3]
```diff
@@ -44,7 +44,11 @@ _All talks will be available live for those with online-only tickets. They will
 -   Eliana Rosselli - An approach to lightweight tenancy management using Django Rest Framework
 -   Elizabeth Christensen ([twitter](https://twitter.com/sqlliz)) - Postgres Performance: From Slow to Pro
 -   Ernesto Rico Schmidt ([mastodon](https://hachyderm.io/@eigenwijsje)) - The evolution of a Django Website into a radio automation back-end
+<<<<<<< HEAD
 -   Felipe ([twitter](https://twitter.com/felipedemorais_)) - AfroPython: Using Django to change black people life in Brazil
+=======
+-   Felipe de Morais ([twitter](https://twitter.com/felipedemorais_)) - AfroPython: Using Django to change black people life in Brazil
+>>>>>>> origin/main
```
[the remark this answers is not in the document]
Comment (drewbrew) [reply _posts/2023-07-10-announcing-lineup.md]: argh, I thought I had resolved that. Fixed

**Review by jefftriplett — APPROVED** [review]

**Review by jefftriplett — APPROVED** [review]
I'll be honest with you, I'm going to check 97 files but I'm confident we can quickly fix anything that looks off.

**Review by jefftriplett — APPROVED** [review]

**Comment by drewbrew** [comment]
schedule

Preview for anyone who is interested

**Pull request merged**